scenarios/smf-failure.xml

```diff
@@ -7,32 +7,32 @@
     <group name="Smf-Failure">
         <group name="SF-Setup-Fwd-1">
             <step name="SF-Create-Session-1"
-                  exec="mock-smf-cmd create"/>
+                  exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c create --count 1 --baseID 1 --ue-pool 17.0.0.0/24 --nb-addr 140.0.100.1"/>
             <step name="SF-Modify-Session-1" requires="^" delay="1"
-                  exec="mock-smf-cmd modify"/>
+                  exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c modify --count 1 --baseID 1 --ue-pool 17.0.0.0/24 --nb-addr 140.0.100.1"/>
         </group>
 
         <step name="SF-Check-Flows-1" requires="SF-Setup-Fwd-1" delay="2"
               exec="onos-cli-grep ${OCI} up4:read-flows 'UL flows=1, DL flows=1'"/>
 
         <step name="SF-Kill-Smf" requires="^"
-              exec="mock-smf-cmd kill"/>
+              exec="${DOCKER_COMPOSE_CMD} kill mock-smf"/>
 
         <step name="SF-Check-Flows-2" requires="^" delay="10"
               exec="onos-cli-grep ${OCI} up4:read-flows 'UL flows=0, DL flows=0'"/>
 
-        <group name="SF-Revive-Smf" requires="~SF-Check-Flows-2">
-            <step name="SF-Start-Smf"
-                  exec="mock-smf-cmd start"/>
-            <step name="SF-Associate-Smf" requires="^" delay="1"
-                  exec="mock-smf-cmd associate"/>
-        </group>
+      <step name="SF-Revive-Smf-1" requires="SF-Check-Flows-2" delay="1"
```

Comment:
Can you group these 3 steps into a separate group as before? Also, we should use a requires with ~ on `SF-Check-Flows-2`. In this way we revive the SMF even if we failed that step.